src/main/java/seedu/todo/commons/util/DateUtil.java

```diff
@@ -197,5 +218,80 @@ public static LocalDateTime parseDateTime(String dateTimeString) {
         DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss");
         return LocalDateTime.parse(dateTimeString, formatter);
     }
+    
+    /*
+     * Check a LocalDateTime if the time is the same as the current time
+     * 
+     * @param date
+     * @return true if it is not the same as current time, false if it is the same as current time 
+     */
+    public static boolean checkIfTimeExist(LocalDateTime date) {
+        LocalDateTime currentTime = LocalDateTime.now();
+        return currentTime.getHour() != date.getHour() || currentTime.getMinute() != date.getMinute();
+    }
+    
+    /*
+     * @@author A0139922Y
+     * Check a LocalDateTime if the date is the same as the current date
+     * 
+     * @param date
+     * @return true if it is not the same as current date, false if it is the same as current date 
+     */
+    public static boolean checkIfDateExist(LocalDateTime date) {
+        LocalDateTime currentDate = LocalDateTime.now();
+        return currentDate.getDayOfYear() != date.getDayOfYear() || currentDate.getMonth() != date.getMonth() || 
+                currentDate.getYear() != date.getYear();
+    }
+    
+    /*
+     * To be used to parse natural date into LocalDateTime 
+     * @parser Natty
+     * 
```

Comment:
Please remove this method, it has been extracted into DateParser. And you're claiming credit for my method.